# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/vim/perl-name-clash.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This is a temporary rename since a global variable name is now also used as
as name inside an enum in perl ...

  - Rene Rebe <rene@rocklinux.org>

--- vim62/src/globals.h.orig	2003-09-30 16:51:07.000000000 +0200
+++ vim62/src/globals.h	2003-09-30 16:52:00.000000000 +0200
@@ -1253,7 +1253,7 @@
 EXTERN char_u e_nowrtmsg[]	INIT(=N_("E37: No write since last change (add ! to override)"));
 EXTERN char_u e_null[]		INIT(=N_("E38: Null argument"));
 #ifdef FEAT_DIGRAPHS
-EXTERN char_u e_number[]	INIT(=N_("E39: Number expected"));
+EXTERN char_u e_number_exp[]	INIT(=N_("E39: Number expected"));
 #endif
 #ifdef FEAT_QUICKFIX
 EXTERN char_u e_openerrf[]	INIT(=N_("E40: Can't open errorfile %s"));
--- vim62/src/digraph.c.orig	2003-09-30 16:51:28.000000000 +0200
+++ vim62/src/digraph.c	2003-09-30 16:51:48.000000000 +0200
@@ -2175,7 +2175,7 @@
 	str = skipwhite(str);
 	if (!isdigit(*str))
 	{
-	    EMSG(_(e_number));
+	    EMSG(_(e_number_exp));
 	    return;
 	}
 	n = getdigits(&str);
